pkg/kubectl/cmd/testdata/edit/testcase-edit-from-empty/test.yaml

```diff
@@ -0,0 +1,17 @@
+description: add a testcase description
+mode: edit
+args:
+- configmap
+namespace: "edit-test"
+expectedStderr:
+- Edit cancelled, no objects found.
+expectedExitCode: 1
+steps:
+- type: request
+  expectedMethod: GET
+  expectedPath: /api/v1/namespaces/edit-test/configmaps
+  expectedInput: 0.request
+  resultingStatusCode: 200
+  resultingOutput: 0.response
+- type: edit
+  expectedInput: 1.original
```

Comment:
a step with `type: edit` means that the editor is still being invoked with an empty list, which is not what we want.